src/main/java/se/sundsvall/messageexchange/api/model/Identifier.java

```diff
@@ -0,0 +1,63 @@
+package se.sundsvall.messageexchange.api.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.Objects;
+
+@Schema(description = "Represents a participant in a conversation, including their type and value.")
+public class Identifier {
+	@Schema(description = "The type of the participant, e.g., adAccount or partyId", example = "user")
```

Comment:
`@Pattern(regexp = "^(adAccount|partyId)$", message = "Type must be 'adAccount' or 'partyId'")` ?